### ISSUE TITLE
Do `Prompt files` actually work?

### ISSUE THREAD

**Issue by alanwilter** [continuedev/continue#2342]
### Relevant environment info

```Markdown
OS: macOS
Continue version: v0.9.211
IDE version:  VSCode  1.93.1
Model: GPT-4o
```

### Description

If I try e.g. https://github.com/continuedev/prompt-file-examples/blob/main/unit-tests.prompt it actually crashes this feature and slash prompts don't work anymore.
So I removed.
I'm trying to do what's shown in the image, which works beautifully!

<img width="525" alt="Screenshot 2024-09-20 at 09 50 36" src="https://github.com/user-attachments/assets/6cff1920-dca3-4bb7-95c5-3e40e20407cd">

Therefore I created `pytest.prompt` with:
```
<system>
You are an expert programmer
</system>

{{{ input }}}

Write unit tests for the above selected code, following each of these instructions:
- Use pytest
- I have pytest already setup, so need to tell me how to install it
- Properly set up and tear down if needed
- Include important edge cases
- The tests should be complete and sophisticated
- Just give the code, no need to explain
- Don't explain how to set up pytest or how to use it
- Add typings according to python 3.9 convention, so no need to use `from typing import` etc.
- Add one line docstring explaining what the test does
```
and tried as seen in the second image.

![image](https://github.com/user-attachments/assets/d823c9ee-f7f4-4767-9c5b-3f2147c109dc)

But it does not work like in my first image.

Using GPT-4o, does do what was asked.
What am I missing?

**Comment by RomneyDa** [comment]
@alanwilter using your prompt and function definition with gpt-4o it seems to work. Can you confirm that your pytest.prompt file was properly saved?

/.prompts/pytest.prompt
```mdx
<system>
You are an expert programmer
</system>

{{{ input }}}

Write unit tests for the above selected code, following each of these instructions:
- Use pytest
- I have pytest already setup, so need to tell me how to install it
- Properly set up and tear down if needed
- Include important edge cases
- The tests should be complete and sophisticated
- Just give the code, no need to explain
- Don't explain how to set up pytest or how to use it
- Add typings according to python 3.9 convention, so no need to use `from typing import` etc.
- Add one line docstring explaining what the test does
```

```py
def sort_array(xs: np.ndarray, n: int) -> np.ndarray:
    return xs[xs[:, n].argsort()]
```

- highlighted function definition, cmd + L to pull into chat
- type /pytest
- go

output (more or less matches your expected output):
<img width="419" alt="image" src="https://github.com/user-attachments/assets/de3c57b4-912b-4c3d-af7b-87f6fe59bf77">

continue v0.8.52

**Comment by RomneyDa** [comment]
@alanwilter you're right, switched to prerelease while keeping model prompt etc. the same and experienced same thing.

**Comment by callum-gander** [comment]
Same on my end, neither release or pre-release works with prompts. It won't add them as custom slash commands

**Comment by softwareliberationarmy** [comment]
Same behavior on Windows 11 with vs code, version 0.8.55. 

* Connected to ollama, I attempted to "Build a custom prompt" from the chat. 
* The system created a new .prompts folder in my workspace. 
* Then I copied the [code smells prompt](https://github.com/continuedev/prompt-file-examples/blob/main/code-smells.prompt) into the file and renamed it 'code-smells.prompt'. 
* Then restarted VS code and suddenly the Continue extension stopped working. 
* Tried restarting and tweaking the config.json file several different ways, nothing fixed it.
* It didn't start working again until I deleted the .prompts folder altogether and restarted vs code again.

**Comment by abalesk** [comment]
Somehow, it didn't like `---` lines in the prompts files; omitting them helped to return functionality (v0.8.55).

**Comment by ctwardy** [comment]
Lots changing in prompts recently, including moving slash commands to @ prompts. 

I have some @ prompts working but it's a bit flaky.
* Best if in the global `.continue\prompts` folder. 
* That folder isn't monitored: changes require re-opening the VSCode window.
* Limited support for calling built-in @ prompts in the prompt file. Right now @ repo-map breaks things for me. 

"Breaks" leads to opaque errors like "Cannot read properties of undefined" on initial Continue load (load project in new window).  

<img width="802" alt="image" src="https://github.com/user-attachments/assets/625c5304-acf0-4d0a-862a-d5d4234d278b" />
